cfgBuilder/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+/*
+Builds the configuration files for the chainbridge application.
+*/
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var app = cli.NewApp()
+
+func init() {
+	app.Action = run
+	app.Copyright = "Copyright 2019 ChainSafe Systems Authors"
+	app.Name = "cfgBuilder"
+	app.Usage = "cfgBuilder [source] [destination]"
+	app.Author = "ChainSafe Systems 2019"
```

Comment:
[Author field has been depreciated](https://pkg.go.dev/github.com/urfave/cli?tab=doc#App). Should use `app.Authors` and assign an Author struct to it.

```
type Author struct {
	Name  string // The Authors name
	Email string // The Authors email
}
```

Reply:
Good catch! The error in CI is actually an issue with go mods, but it does look like we should update the CLI at some point https://pkg.go.dev/mod/github.com/urfave/cli/v2@v2.2.0